proposals/blogProposal.md

```diff
@@ -0,0 +1,9 @@
+The Web of Things standardization activities are now illustrated through a new animation video that we have been working on for the last few months. You can see how a web developer can implement Internet of Things solutions while being interoperable and not locked into frameworks. Additionally, the W3C WoT allows the experience of the developers to be transferred to different application domains such as agriculture, building or industrial automation, smart home and more!
```

Comment:
Delete "that we have been working on for the past few months."   Not relevant, and it makes us look slow.  "You can see how" -> "The animation shows how", "allows the experience of the developers" -> "allows developer experience and skills"